Tools/eslint-config-cesium/index.js

```diff
@@ -1,4 +1,7 @@
-{
+/*eslint-env node*/
```

Comment:
This shouldn't be needed if you set the top level .eslintrc for this directory to be node, right?